Cargo.toml

```diff
@@ -657,6 +659,7 @@ sinks-metrics = [
 ]
 
 sinks-amqp = ["lapin"]
+sinks-appsignal = ["dep:tonic", "protobuf-build"]
```

Comment:
We're not using tonic as the HTTP client, but I did have to add it as a dependency for the protobuf to be available if compiled/tested with:

```
cargo test --lib --no-default-features --features sinks-appsignal sinks::appsignal
```